src/lib/constants.js

```diff
@@ -33,4 +36,33 @@ module.exports = Object.freeze({
   ],
   ENUM_POST_TYPE: ['Request', 'Donate'],
   ENUM_POST_STATUS: ['Published', 'Verified'],
+
+  ZIP_REGEX: new RegExp(/^\d{5}(?:[-\s]\d{4})?$/),
+  PASSWORD_REGEX: new RegExp(
+    /^(?=.*?[A-Z])(?=(.*[a-z]){1,})(?=(.*[\d]){1,})(?=(.*[\W]){1,})(?!.*\s).{7,}$/
+  ),
+  NAME_REGEX: new RegExp(/^$|\s+/),
+  EMAIL_REGEX: new RegExp(
+    /^[a-zA-Z0-9\-_.]+@[a-z]+\.([a-z]{2,3})+(\.[a-z]{2,3})?$/
+  ),
+  USERNAME_REGEX: new RegExp(/^[a-zA-Z0-9\-_.]{2,20}$/),
+  PHONE_NUMBER_REGEX: new RegExp(/^[+]?[0-9]{1,3}[-\s]?[0-9]{1,3}[0-9]{4,9}$/),
+
```

Comment:
extra new line

Reply:
No extra lines allowed. I had `// models' constants end` for that, but removed upon request. Added empty comment lines now instead.